tools/respecDocWriter.js

```diff
@@ -186,16 +186,24 @@ async function evaluateHTML() {
         ? [123456789, 0, 0]
         : window.respecVersion.split(".").map(str => parseInt(str, 10));
     if (major < 20 || (major === 20 && minor < 10)) {
+      console.warn(
+        "👴🏽  Ye Olde ReSpec version detected! Please update to 20.10.0 or above. " +
+          `Your version: ${window.respecVersion}.`
+      );
       // Document references an older version of ReSpec that does not yet
       // have the "core/exporter" module. Try with the old "ui/save-html"
       // module.
-      const { exportDocument } = await new Promise(resolve => {
-        require(["ui/save-html"], resolve);
+      const { exportDocument } = await new Promise((resolve, reject) => {
+        require(["ui/save-html"], resolve, err => {
+          reject(new Error(err.message));
```

Comment:
maybe we can reject directly without creating a new `Error` object?

Reply:
I got w better stack in the browser by wrapping it. The error passed by requirejs didn’t seem very helpful. We want these lines to show up in the stack.